sdk/communication/Azure.Communication.MediaComposition/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Release History
+
+## 2021-12-31-preview
```

Comment:
The guideline for changelog is `## <version> (release date)`. 
So this should be `## 1.0.0-beta.1 (Unreleased)` for now